### ISSUE TITLE
Cannot Find in plug-in repo through octoprint and receiving this error when attempting to install manually.

### ISSUE THREAD

**Issue by jk257** [edekeijzer/OctoPrint-WebDavBackup#13]
ERROR: Command errored out with exit status 1:
command: /home/pi/oprint/bin/python2 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-aiJiO3/lxml/setup.py'"'"'; __file__='"'"'/tmp/pip-install-aiJiO3/lxml/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-aiJiO3/lxml/pip-egg-info
cwd: /tmp/pip-install-aiJiO3/lxml/
Complete output (3 lines):
Building lxml version 4.6.2.
Building without Cython.
Error: Please make sure the libxml2 and libxslt development packages are installed.
----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Error!
Could not parse output from pip, see plugin_pluginmanager_console.log for generated output

**Comment by edekeijzer** [comment]
You're running Python 2, which isn't supported. Therefore you're unable to see it in the repo.
See [this blog post](https://octoprint.org/blog/2020/09/10/upgrade-to-py3/) on how to upgrade to Python 3.

**Comment by jk257** [comment]
Thank you, I upgraded to Python 3 and now I can see the plug-in in the repo and was able to click it to install. It runs through the install, says it was successful, then prompts to reboot. When I do and the pi reboots and I log back in I cant find the plug in. When I look at plug in manager it shows its installed but then I look at the repo to install and the plugin is still listed to be installed as if I never did anything. I've tried multiple times. I've been able to install other plug-ins today so not sure what's going on now.
